css/style.css

```diff
@@ -381,6 +381,22 @@ header {
   border: none;
   padding: 10px 68px;
   line-height: 24px;
+  cursor: pointer;
+}
+
+.email-error {
+  border: 1px solid red;
+}
+
+.email-error::before {
+  content: "Please write your email address in small letters.";
+  color: red;
+  font-size: 12px;
+  margin-top: 0;
+}
```

Comment:
Please ensure that the error message shows up close to the "submit" button according to the project requirement.